bots/BotLib/hero_dragon_knight.lua

```diff
@@ -87,26 +85,23 @@ sRoleItemsBuyList['pos_3'] = {
 	"item_double_branches",
 	"item_quelling_blade",
 	"item_gauntlets",
-	"item_circlet",
-	"item_magic_wand",
 
 	"item_bracer",
 	"item_boots",
+	"item_magic_wand",
 	"item_power_treads",
-	"item_hand_of_midas",
+	"item_orchid",
 	"item_blink",
+	"item_sange_and_yasha",--
 	"item_ultimate_scepter",
 	"item_black_king_bar",--
+	"item_bloodthorn	",
 	"item_assault",--
-	"item_octarine_core",--
-	"item_orchid",
-	"item_moon_shard",
-	"item_bloodthorn",--
-	"item_ultimate_scepter_2",
 	"item_travel_boots",
+	"item_ultimate_scepter_2",
 	"item_overwhelming_blink",--
 	"item_travel_boots_2",--
-
+	"item_moon_shard",
```

Comment:
there is a bug in Valve provided item management where the late moon shard will stay in the backpack and bot wont use it at all. This is the only reason I made bots purchase moon shard a bit early.
You can test it out.